core/tests/protractor/collections.js

```diff
@@ -71,6 +75,33 @@ describe('Collections', function() {
     users.logout();
   });
 
+  it('searches explorations in collection editor', function() {
+    users.login('alice@collections.com');
+    browser.get(general.SERVER_URL_PREFIX);
+    var dropdown = element(by.css('.protractor-test-profile-dropdown'));
+    browser.actions().mouseMove(dropdown).perform();
+    dropdown.element(by.css('.protractor-test-dashboard-link')).click();
+    browser.waitForAngular();
+    element(by.css('.protractor-test-create-activity')).click();
+    // Create new collection.
+    element(by.css('.protractor-test-create-collection')).click();
+    browser.waitForAngular();
+    // Search and add existing explorations.
```

Comment:
You know, instead of adding an entirely new test for this, it might be better (or at least faster during Travis runs) to just augment the preceding test. What do you think? Many of the actions seem the same.